src/include/migraphx/op/if_op.hpp

```diff
@@ -71,7 +71,7 @@ struct if_op
         std::unordered_map<std::string, argument> params;
 
         std::set<std::string> pnames;
-        for(const auto& smod : mods)
+        for(const const_module_ref& smod : mods)
```

Comment:
Why does it have two `const` ?

Reply:
One is const for the pointer and the other is for the data the pointer points to. I guess it doesnt really make sense to use a const ref here since `const_module_ref` is just a pointer.